docs/source/integrations/plugins-event-reference.mdx

```diff
@@ -487,6 +487,8 @@ if the GraphQL operation encounters one or more errors.
 
 (If the operation uses [incremental delivery](../workflow/requests#incremental-delivery-experimental) directives such as `@defer`, `willSendResponse` is called before the *initial* payload is sent; you can use `willSendSubsequentPayload` to find out when more payloads will be sent.)
 
+The `willSendResponse` event is also where you can mutate the GraphQL response in-place. For example, you can add values to the `extensions` object or redact information in the `error` blocks.
```

Comment:
Shouldn't the latter be handled by [formatError](https://www.apollographql.com/docs/apollo-server/data/errors/#for-client-responses)?

Reply:
Good callout. I will update to reference that section